engine/skiplist.cpp

```diff
@@ -13,80 +13,15 @@
 
 namespace KVDK_NAMESPACE {
 
+std::unordered_map<std::string, CompContext> &GetCollectionCompFuncMap() {
```

Comment:
This shouldn't be singleton, it should be per kvdk instance owned, as two instance maybe have two sorted collections with the same name

Reply:
I know